Jenkinsfile_CNP

```diff
@@ -0,0 +1,17 @@
+#!groovy
+
+@Library("Infrastructure")
+
+def type = "java"          // supports "java", "nodejs" and "angular"
+
+def product = "idam-java-client"
+
+def component = "idam-java-client" // must match infrastructure module name
+
+withPipeline(type, product, component) {
```

Comment:
you probably don't want to use withPipeline it's not designed for this,

you'll need to copy the required code in to the jenkinsfile